backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala

```diff
@@ -380,6 +380,26 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
       checkOperatorMatch[ProjectExecTransformer])
   }
 
+  test("test 'function current_date'") {
+    runSql("select current_date() from range(10)")(checkOperatorMatch[ProjectExecTransformer])
+  }
+
+  test("test 'function current_timestamp'") {
```

Comment:
Not sure could this use `runQueryAndCompare`, because `current_timestamp` is volatitle

Reply:
Could you please confirm whether an `Alias` project is produced?

Reply:
U r right.